zproject/settings.py

```diff
@@ -1079,3 +1079,5 @@ def get_secret(key):
 PROFILE_ALL_REQUESTS = False
 
 CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+
+USE_SOCKET = True
```

Comment:
I was going to call it that actually, I was going off of another comment on the issue. I'll change it now.